desktop/plugins/network/index.tsx

```diff
@@ -385,6 +394,67 @@ export function plugin(client: PluginClient<Events, Methods>) {
 
         informClientMockChange(routes.get());
       },
+      importRoutes() {
+        const options: OpenDialogOptions = {
+          properties: ['openFile'],
+          filters: [{extensions: ['json'], name: 'Flipper Route Files'}],
+          title: 'Import Routes',
+        };
+        remote.dialog.showOpenDialog(options).then((result) => {
+          const filePaths = result.filePaths;
+          if (filePaths.length > 0) {
+            fs.readFile(filePaths[0], 'utf8', (err, data) => {
+              if (err) {
+                console.error(err);
+                return;
+              }
+              const importedRoutes = JSON.parse(data);
+              importedRoutes?.forEach((importedRoute: Route) => {
+                if (importedRoute != null) {
+                  const newNextRouteId = nextRouteId.get();
+                  routes.update((draft) => {
+                    draft[newNextRouteId.toString()] = {
+                      requestUrl: importedRoute.requestUrl,
+                      requestMethod: importedRoute.requestMethod,
+                      responseData: importedRoute.responseData as string,
+                      responseHeaders: importedRoute.responseHeaders,
+                      responseStatus: importedRoute.responseStatus,
+                    };
+                  });
+                  nextRouteId.set(newNextRouteId + 1);
+                }
+              });
+              informClientMockChange(routes.get());
+            });
+          }
+        });
+      },
+      exportRoutes() {
+        remote.dialog
+          .showSaveDialog(
+            // @ts-ignore This appears to work but isn't allowed by the types
+            null,
+            {
+              title: 'Export Routes',
+              defaultPath: 'NetworkPluginRoutesExport.json',
+            },
+          )
+          .then(async (result: electron.SaveDialogReturnValue) => {
+            const file = result.filePath;
+            if (!file) {
+              return;
+            }
+            fs.writeFileSync(
+              file,
+              JSON.stringify(routes.get(), null, 2),
+              'utf8',
+            );
+          });
```

Comment:
NIT: a message after success would be nice :)

Reply:
Can I use "message()" like you suggest in your other comment or is there another technique?  Thanks.

Reply:
Yeah, message.info should be great for this :)

Reply:
Completed.  Here's the new screenshot:

![image](https://user-images.githubusercontent.com/337874/105763561-8c727880-5f1b-11eb-9adc-ea95f2158be7.png)